Cannonball/ThemeChooserViewController.swift

```diff
@@ -87,19 +111,19 @@ class ThemeChooserViewController: UITableViewController {
         )
     }
 
-    override func viewDidAppear(animated: Bool) {
+    override func viewDidAppear(_ animated: Bool) {
         super.viewDidAppear(animated)
 
         // Make sure the navigation bar is translucent.
-        navigationController?.navigationBar.translucent = true
+        navigationController?.navigationBar.isTranslucent = true
     }
 
-    override func viewWillDisappear(animated: Bool) {
+    override func viewWillDisappear(_ animated: Bool) {
         super.viewWillDisappear(animated)
 
         // Move the logo view off screen if a new controller was pushed.
         if navigationController?.viewControllers.count > 1 {
```

Comment:
Change to `if let vcCount = navigationController?.viewControllers.count, vcCount > 1 {` so we can remove the `FIXME` for comparison operators with optionals.